helpers/check_format.sh

```diff
@@ -45,7 +45,7 @@ validate_python() {
     # Initialize FILES_TO_LINT to empty string
     FILES_TO_LINT=""
 
-    (cd $FOLDER && flake8 --exclude=.git,__pycache__,.venv,venv --ignore=E501,W5)
+    (cd $FOLDER && flake8 --exclude=.git,__pycache__,.venv,venv --select=E9,F)
```

Comment:
nit: It would be valuable to also have [complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity) checks provided by [mccabe plugin](https://github.com/PyCQA/mccabe) by default [uses C class of errors](https://flake8.pycqa.org/en/latest/glossary.html#term-error-code).
```suggestion
    (cd $FOLDER && flake8 --exclude=.git,__pycache__,.venv,venv --select=E9,F,C)
```

Reply:
Included complexity check as well